modules/codegen/test/src/smithy4s/codegen/internals/TestUtils.scala

```diff
@@ -101,4 +102,29 @@ object TestUtils {
       .unwrap()
   }
 
+  def loadAndValidateModel(namespaces: String*): Model = {
```

Comment:
Using `loadModel`, which disabled validations, led to unpredictable results - like shapes not being correctly handled by walker, incorrect output/input shapes (i.e. allowing booleans to be naked inputs), etc.

I'm not sure what the original rationale for this was, but it affected the behaviour of tests so severely that I decided to stop guessing what went wrong and just fix the test specs instead.